src/NetworkFunction/custom/Set-AzNetworkFunctionCollectorPolicy.ps1

```diff
@@ -0,0 +1,210 @@
+
+# ----------------------------------------------------------------------------------
```

Comment:
Why we need to customize Set-Az* cmdlet? We usually hide Set cmdlet from customer.

Reply:
Removed the set cmdlets